static/js/poll_widget.js

```diff
@@ -32,6 +32,7 @@ export function activate({
 
     function render_question() {
         const question = poll_data.get_question();
+        const rendered_question = question + `<i class="fa fa-pencil poll-edit-question"></i>`;
```

Comment:
I don't think we want to move this logic out of the hbs template and into JavaScript... is there a reason why you believe this is the right fix, other than the fact that it works?

Reply:
Javascript was rendering text into the heading so even if put the icon in the heading by changing hbs it would get replaced by the text. So to put the icon just after the end of text i thought of rendering html into the heading.

Reply:
Why not keep it in a separate element but `display: inline`?

Reply:
Okay got it, thank you. I have made the suggested change